lib/components/form/mode-selector.js

```diff
@@ -9,25 +9,44 @@ class ModeSelector extends Component {
     location: PropTypes.object,
     label: PropTypes.string,
     setLocation: PropTypes.func,
+    showLabel: PropTypes.boolean,
     type: PropTypes.string // replace with locationType?
   }
+
   _onChange = (evt) => {
     console.log(evt.target.value)
     this.props.setMode(evt.target.value)
   }
+
+  _getDisplayText (mode) {
+    switch (mode) {
+      case 'TRANSIT,WALK': return 'Walk to Transit'
+      case 'TRANSIT,BICYCLE': return 'Bike to Transit'
+      case 'WALK': return 'Walk Only'
+      case 'BICYCLE': return 'Bike Only'
+    }
+    return mode
+  }
+
   render () {
     const { config, mode } = this.props
+    const label = this.props.label || 'Mode:'
```

Comment:
I'm not sure if we've started in otp-rr yet, but for default props maybe we should start using `static defaultProps = {label: 'Mode'}` https://babeljs.io/blog/2015/06/07/react-on-es6-plus.